tiltak/src/main/kotlin/no/nav/amt/tiltak/nav_enhet/NavEnhetRepository.kt

```diff
@@ -21,47 +20,29 @@ open class NavEnhetRepository(
 		)
 	}
 
-	fun upsert(enhetId: String, navn: String): NavEnhetDbo {
+	fun insert(id: UUID, enhetId: String, navn: String) {
```

Comment:
Burde man gått for command-patternet her også?

Reply:
Eller er det såpass lite og kommer aldri til å bli så stort at det er nødvendig?

Reply:
Tror ikke det blir noen flere felter, men vi har brukt patternet en del andre steder, så argumentet for å bruke det her blir vel mest for at det blir likt i hele appen.

Reply:
Tenker jeg legger det til 👍